presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -440,4 +447,48 @@ private static double percentage(double count, double total)
         }
         return min(100, (count * 100.0) / total);
     }
+
+    private static class ConsoleWarningsPrinter
+            extends AbstractWarningsPrinter
+    {
+        private final ConsolePrinter console;
+
+        public ConsoleWarningsPrinter(ConsolePrinter console)
+        {
+            this.console = requireNonNull(console, "console is null");
+        }
+
+        @Override
+        public void print(List<PrestoWarning> warnings)
+        {
+            requireNonNull(warnings, "warnings is null");
+
+            console.reprintLine("");
+
+            int warningsSize = warnings.size();
+            int warningsRenderedCount = getProcessedWarningsCount();
+            int end = Math.min(warningsSize, warningsRenderedCount + StatusPrinter.WARNING_LINES);
```

Comment:
What does `WARNING_LINES` do? Can you please rename it to better signal its intent?